docker/docker-compose.yml

```diff
@@ -226,12 +257,18 @@ services:
       PGRST_JWT_SECRET: ${JWT_SECRET}
       DATABASE_URL: postgres://supabase_storage_admin:${POSTGRES_PASSWORD}@${POSTGRES_HOST}:${POSTGRES_PORT}/${POSTGRES_DB}
       FILE_SIZE_LIMIT: 52428800
-      STORAGE_BACKEND: file
+      STORAGE_BACKEND: s3
+      GLOBAL_S3_BUCKET: stub
+      GLOBAL_S3_ENDPOINT: http://minio:9000
+      GLOBAL_S3_PROTOCOL: http
+      GLOBAL_S3_FORCE_PATH_STYLE: true
```

Comment:
Does storage fail gracefully if user didn't create a s3 bucket? If not, we probably need to update the self-hosted guide together with these changes.

In addition, we need to document the upgrade path for existing self-hosted users with locally mounted volumes. Seems like quite a bit of work to get this through.